webgme/webgme-blob.d.ts

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
It would be better to just have 3 files: `index.d.ts`, `webgme-tests.ts`, and `tsconfig.json`.